hack/test-extended.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+source ${OS_ROOT}/hack/util.sh
+source ${OS_ROOT}/hack/common.sh
+
+# TODO: Randomize these ports
```

Comment:
Yes, that was my thinking... We have random ports when you start the server using utils/server.go (StartAllInOneServer?)... Problem is that having random ports also means we need to have bash function that will scan if the ports is free... I don't want to add this now... I think I will add more reasoning to this comment